src/pages/connections/farmlands.mdx

```diff
@@ -10,43 +11,43 @@ In partnership with Centrapay, Farmlands has two new digital solutions that enab
 Your business can either fully integrate the Farmlands solution to your retail point of sale system, or access a secure web portal when accepting Farmlands Card as a form of payment for a sale anytime, anywhere.
 
 <center>
-```mermaid
-%%{
-  init: {
-    'theme': 'base',
-    'themeVariables': {
-      'secondaryColor': '#f8f8f8',
-      'tertiaryColor': '#f8f8f8'
-    }
-  }
-}%%
+  <div  style="max-width: 800px">
```

Comment:
Is there no way of doing this in the layout?

Reply:
No, but i'll do this in a seperate PR